{"name": "test_invalid_login_shows_error", "status": "passed", "description": "Invalid login should display an error message.", "start": 1757946795483, "stop": 1757946818737, "uuid": "60c98256-cf8a-4df9-b77e-cd0462d6bd3a", "historyId": "ef3403fe28d4b542f8ab987512d4fce5", "testCaseId": "100c00eabc4ddcc756ea44c0b5d967b8", "fullName": "tests.test_login#test_invalid_login_shows_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DB-L-174"}, {"name": "thread", "value": "1824-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}